modules/get-started/pages/whats-new.adoc

```diff
@@ -19,6 +19,10 @@ xref:manage:security/authorization/rbac.adoc[Role-based access control] (RBAC) p
 
 * xref:develop:data-transforms/run-transforms.adoc#view-data-transform-logs[Data transform logs] are now stored in an internally-managed topic and are accessible via `rpk` and the Redpanda Console.
 
+== Topic-aware partition balancing 
+
+Redpanda now prioritizes xref:manage:cluster-maintenance/cluster-balancing.adoc[balancing a topic's partition count on each broker] while it's balancing the cluster's overall partition count. Because different topics in a cluster can have vastly different load profiles, this better distributes the workload evenly across brokers. This feature is enabled by default with the `partition_autobalancing_topic_aware` property. After upgrading to 24.1, new topics are created in a topic-aware fashion, and the next partition rebalance (triggered by adding new brokers or by calling the `/v1/partitions/rebalance` Admin API endpoint) evens out the partition counts on each broker.
```

Comment:
' a topic's partition replica count' (technically these are replicas not entire partitions being balanced)

Reply:
" new topics are distributed in a topic-aware fashion". (distribution of replicas is the thing here)

Reply:
" and the next partition rebalance (triggered by adding new brokers or by calling the `/v1/partitions/rebalance` Admin API endpoint) evens out the partition counts on each broker."

This is misleading @micheleRP because all continuous data balancing (triggered by disk fullness not using the Admin API directly) _also_ would be topic-aware

cc @ztlpn

Reply:
Thanks, @mattschumpert! Please review the [updated section](https://deploy-preview-416--redpanda-docs-preview.netlify.app/24.1/manage/cluster-maintenance/cluster-balancing/).